apps/web/pages/team/[slug].tsx

```diff
@@ -34,15 +34,12 @@ function TeamPage({ team }: TeamPageProps) {
   const telemetry = useTelemetry();
 
   useEffect(() => {
-    telemetry.withJitsu((jitsu) =>
-      jitsu.track(
-        telemetryEventTypes.pageView,
-        collectPageParameters("/team/[slug]", {
-          isTeamBooking: true,
-        })
-      )
+    telemetry.event(
```

Comment:
It could. I kept it for compatibility. However, I don't think duplicated events is a problem. Do you could sum(page_views) as opposed to distinct page views?